modules/core/04-channel/keeper/upgrade.go

```diff
@@ -802,9 +802,18 @@ func (k Keeper) constructProposedUpgrade(ctx sdk.Context, portID, channelID stri
 	}, nil
 }
 
-// AbortUpgrade will restore the channel state and flush status to their pre-upgrade state so that upgrade is aborted.
+// MustAbortUpgrade will restore the channel state and flush status to their pre-upgrade state so that upgrade is aborted.
 // any unnecessary state is deleted. An error receipt is written, and the OnChanUpgradeRestore callback is called.
```

Comment:
I think both godocs can be updated to remove the "and the OnChanUpgradeRestore callback is called"